app/(note)/_components/noteContent/editor/linkToolbarButton/LinkToolbarButton.tsx

```
@@ -5,14 +5,19 @@ import React from 'react';
 import { useComponentsContext } from '@blocknote/react';
 
 import { LinkIcon } from '@/components/icons';
+import { useModalStore } from '@/provider/store-provider';
 
-interface Props {
-  onClick: () => void;
-}
+interface Props {}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)